core.py

```diff
@@ -220,8 +213,6 @@ def folder_reader(path):
                                 if decompressed:
                                     queue.put(decompressed)
 
-                                f.close()
```

Comment:
Files are closed by the exit part of context manager of file objects (see http://book.pythontips.com/en/latest/context_managers.html#context-managers).

Reply:
You're right, thank you for pointing that!